ethereum/spec/src/test/java/tech/pegasys/teku/spec/config/SpecConfigFuluTest.java

```diff
@@ -80,12 +79,41 @@ public void shouldOverrideBlobRelatedValuesValues() {
     final SpecConfigElectra electraConfig =
         specConfigAndParent.forMilestone(SpecMilestone.ELECTRA).toVersionElectra().orElseThrow();
 
-    final SpecConfigFulu fuluConfig =
-        specConfigAndParent.forMilestone(SpecMilestone.FULU).toVersionFulu().orElseThrow();
-
     assertThat(denebConfig.getMaxBlobsPerBlock()).isEqualTo(4);
     assertThat(electraConfig.getMaxBlobsPerBlock()).isEqualTo(8);
-    assertThat(fuluConfig.getMaxBlobsPerBlock()).isEqualTo(12);
+  }
+
+  @Test
+  public void shouldMatchElectraForkEpochBlobCount() {}
```

Comment:
Forgot?

Reply:
oh yes i think thats covered i should just delete that